### PR TITLE
update to exclude tooling from beachball publishing

### PR DESCRIPTION
**Pull request by awentzel**
Publishing with Beachball is failing because tooling is using `latest` versions.  When beachball attempts to publish, the latest version for tools changes to an actual version, then is unable to bump because that version already exists.

By excluding tools, for now, we can avoid this issue and continue to publish.  @janechu can provide additional details as to the temporary nature of this exclusion.

For exclusion details refer to https://microsoft.github.io/beachball/concepts/groups.html
### 🎫 Issues
* Beachball fails thinking it's publishing to an already existing version

## 📑 Test Plan

Will manually deploy a publish to determine if this in fact resolves our issue.

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: build/releasing/index.ts [hunk 1]
```diff
@@ -6,11 +6,8 @@ export const config: BeachballConfig = {
     groups: [
         {
             name: "Microsoft FAST",
-            include: [
-                "packages/tooling/*",
-                "packages/utilities/*",
-                "packages/web-components/*",
-            ],
+            include: ["packages/utilities/*", "packages/web-components/*"],
+            exclude: ["packages/tooling/*"],
```
Comment (chrisdholt) [inline build/releasing/index.ts:10]: Do we want to exclude all tooling from versioning together? I think we just want to exclude the version which is explicitly pinned to `latest`.

```suggestion
            include: ["packages/tooling/*", "packages/utilities/*", "packages/web-components/*",],
            exclude: ["packages/tooling/fast-tooling"],
```

Comment (chrisdholt) [reply build/releasing/index.ts:10]: @janechu can advise, but I think we just want this to exclude that single package

File: build/releasing/index.ts [hunk 2]
```diff
@@ -6,11 +6,9 @@ export const config: BeachballConfig = {
     groups: [
         {
             name: "Microsoft FAST",
-            include: [
-                "packages/tooling/*",
-                "packages/utilities/*",
-                "packages/web-components/*",
-            ],
+            include: ["packages/utilities/*", "packages/web-components/*"],
```
Comment (nicholasrice) [inline build/releasing/index.ts:9]: I think you want to delete the first `include` field

Comment (awentzel) [reply build/releasing/index.ts]: I guess the web merge doesn't work so well.  Resolved.

**Review by nicholasrice — APPROVED** [review]

**Review by chrisdholt — APPROVED** [review]

**Pull request merged**